solr/contrib/ltr/src/java/org/apache/solr/ltr/interleaving/LTRInterleavingRescorer.java

```diff
@@ -141,7 +141,10 @@ public void scoreFeatures(IndexSearcher indexSearcher,
       }
       for (int i = 0; i < rerankingQueries.length; i++) {
         if (modelWeights[i] != null) {
-          scoreSingleHit(indexSearcher, topN, modelWeights[i], docBase, hitUpto, new ScoreDoc(hit.doc, hit.score, hit.shardIndex), docID, rerankingQueries[i], scorers[i], rerankedPerModel[i]);
+          final ScoreDoc hit_i = new ScoreDoc(hit.doc, hit.score, hit.shardIndex);
```

Comment:
if i remember correctly in Java we are not supposed to use _ as separator, I think CamelCase should be used.
Not a strong opinion anyway, we could rename the variable to something like 'currentHit' or similar, up to you!